test/fixtures/parser/bigquery/select_mixture_of_array_literals.yml

```diff
@@ -34,7 +34,7 @@ file:
             array_literal:
               start_square_bracket: '['
               expression:
-                literal: "'a'"
+                literal: '"a"'
```

Comment:
This manual edit is only included to show the check failing.

Once happy with this PR then let me know and I will revert this change and the test will pass.